intelmq/bots/experts/cut_from_string/expert.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+Cut from string
+"""
+from intelmq.lib.bot import Bot
+
+
+class CutFromStringExpertBot(Bot):
+    string_from_start: int = 1  # 1 - from start, 0 - from end
+    string_for_cut: str = 'www.'
+    field_for_cut: str = 'source.fqdn'
```

Comment:
I think `field_for_cut` should simply be called `field` - there are already some other bots that operate on fields, that use the variable name `field`